src/SFA.DAS.AssessorService.Application.Api/Infrastructure/CompaniesHouseApiClient.cs

```diff
@@ -50,8 +50,14 @@ public async Task<bool> IsCompanyActivelyTrading(string companyNumber)
 
             if (company != null)
             {
-                isTrading = "active".Equals(company.Status, StringComparison.InvariantCultureIgnoreCase) &&
-                            company.DissolvedOn == null && company.IsLiquidated != true;
+                if(company.Status is null && "charitable-incorporated-organisation".Equals(company.Type, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Should this string be a constant?

Reply:
yes but the code below is done in the same way so keeping it consistent ;)